# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/PSocketProcessBridge.ipdl
# Commit: c414d82a5325
# Full Hash: c414d82a532508523d40ce63111cb30b2a1076be
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-02-22 05:26:22
# Regressor Bug: 1521879
# File Overlap Count: 1
# Description:
#   Bug 1521879 - Part 1: IPC-based MediaTransport implementation r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D17273
# ==============================================================================

diff -r 3a182b05e2c1 -r c414d82a5325 netwerk/ipc/PSocketProcessBridge.ipdl
--- a/netwerk/ipc/PSocketProcessBridge.ipdl	Thu Feb 21 19:01:32 2019 +0200
+++ b/netwerk/ipc/PSocketProcessBridge.ipdl	Thu Feb 21 16:42:12 2019 +0000
@@ -5,6 +5,7 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 include protocol PBackground;
+include protocol PMediaTransport;
 
 namespace mozilla {
 namespace net {
@@ -19,9 +20,13 @@
   */
 nested(upto inside_cpow) sync protocol PSocketProcessBridge
 {
+  manages PMediaTransport;
+
 parent:
   async InitBackground(Endpoint<PBackgroundParent> aEndpoint);
 
+  async PMediaTransport();
+
 both:
   async Test();
 };